pyproject.toml

```diff
@@ -70,7 +84,6 @@ exclude = [
     'core/usage_statistics/usage_statistics\.py', # 19
     'core/usage_statistics/util\.py', # 2
     'core/util\.py', # 18
-    'docs/',  # Docs should not be type checked with the rest of the library.
```

Comment:
@anthonyburdi 
I moved this to the ALWAYS exclude section and tried to make be visible.